packages/core/useFullscreen/index.ts

```diff
@@ -93,35 +35,87 @@ export function useFullscreen(
   target?: MaybeElementRef,
   options: UseFullscreenOptions = {},
 ) {
-  const { document = defaultDocument, autoExit = false } = options
-  const targetRef = target || document?.querySelector('html')
+  const {
+    document = defaultDocument,
+    autoExit = false,
+  } = options
+
+  const targetRef = computed(() => unrefElement(target) ?? document?.querySelector('html'))
   const isFullscreen = ref(false)
-  let map: FunctionMap = functionsMap[0]
 
-  const isSupported = useSupported(() => {
-    if (!document) {
-      return false
-    }
-    else {
-      const target = unrefElement(targetRef)
+  const requestMethod = computed<'requestFullscreen' | undefined>(() => {
+    return [
+      'requestFullscreen',
+      'webkitRequestFullscreen',
+      'webkitEnterFullscreen',
+      'webkitEnterFullScreen',
+      'webkitRequestFullScreen',
+      'mozRequestFullScreen',
+      'msRequestFullscreen',
+    ].find(m => (document && m in document) || (targetRef.value && m in targetRef.value)) as any
+  })
+
+  const exitMethod = computed<'exitFullscreen' | undefined>(() => {
+    return [
+      'exitFullscreen',
+      'webkitExitFullscreen',
+      'webkitExitFullScreen',
+      'webkitCancelFullScreen',
+      'mozCancelFullScreen',
+      'msExitFullscreen',
+    ].find(m => (document && m in document) || (targetRef.value && m in targetRef.value)) as any
+  })
+
+  const fullscreenEnabled = computed<'fullscreenEnabled' | undefined>(() => {
+    return [
+      'fullScreen',
```

Comment:
Hello, I'd like to confirm if this property name is really correct and expected. I didn't see the `fullScreen` property but see the [`fullscreen`](https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreen) property from the MDN documentation. 🤔